packages/tools/devtools/devtools-view/src/components/data-visualization/TreeHeader.tsx

```diff
@@ -48,15 +48,15 @@ export function TreeHeader(props: TreeHeaderProps): React.ReactElement {
 	return nodeValue !== undefined ? (
 		<TreeItemLayout style={{ marginLeft: "25px" }}>
 			{`${label}`}
-			<span style={{ color: tokens.colorPaletteRedBorderActive, fontSize: "12px" }}>
+			<span style={{ color: tokens.colorPaletteRedBorderActive, fontSize: "10px" }}>
```

Comment:
These changes will conflict with the tree refactoring I just merged, sorry 😕 I think it should be pretty easy to resolve though.

Reply:
yup! was pretty quick